qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -329,7 +329,7 @@ def write_ceph_exclude(self):
         # These tests will fail or take too much time and will
         # make the test timedout, just skip them for now.
         xfstests_exclude_contents = dedent('''\
-            {c}/001 {g}/003 {g}/075 {g}/538 {g}/531
+            {c}/001 {g}/003 {g}/538 {g}/531 {g}/650 {g}/397 {g}/421 {g}/429 {g}/440 {g}/580 {g}/593 {g}/595
```

Comment:
Also, it would be nicer to wrap this line at 80 characters.

Reply:
Can we add a note about each set of tests in a comment that are excluded?

Reply:
> Can we add a note about each set of tests in a comment that are excluded?

+1